checkov/terraform/checks/resource/github/BranchProtectionReviewNumTwo.py

```diff
@@ -18,7 +19,7 @@ def scan_resource_conf(self, conf: dict[str, list[Any]]) -> CheckResult:
         if conf.get("required_pull_request_reviews") and isinstance(conf.get("required_pull_request_reviews"), list):
             review = conf.get("required_pull_request_reviews")[0]
             if review.get("required_approving_review_count") and isinstance(review.get("required_approving_review_count"),list):
-                count = review.get("required_approving_review_count")[0]
+                count = force_int(review.get("required_approving_review_count")[0])
                 if count >= 2:
```

Comment:
```suggestion
                if count and count >= 2:
```
if `None` comes out of `force_int`